src/pages/LogView/LogWindow/index.tsx

```diff
@@ -24,9 +24,10 @@ const LogWindow: React.FC<LogWindowProps> = ({ logType }) => {
     () => logLines.map((_, index) => index),
     [logLines]
   );
+
   return (
     <Container>
-      <SideBar />
+      {logLines.length !== 0 && <SideBar maxLineNumber={logLines.length - 1} />}
```

Comment:
This is because the `logLines` start out as `-1` length when they haven't finished processing/loading